communication/resources/README.md

```diff
@@ -0,0 +1,82 @@
+# Kubernetes Resources
+
+> A collection of resources organized by medium (e.g. audio, text, video)
+
+## Table of Contents
+
+<!-- vim-markdown-toc GFM -->
+
+- [Contributions](#contributions)
+- [Resources](#resources)
+  - [Audio](#audio)
+  - [Text](#text)
+  - [Video](#video)
+  - [Miscellaneous](#miscellaneous)
+
+<!-- vim-markdown-toc -->
+
+## Contributions
+
+If you would like to contribute to this list, please submit a PR and add `/sig contributor-experience` and `/assign @petermbenjamin`.
+
+The criteria for contributions are simple:
+
+- The resource must be related to Kubernetes.
+- The resource must be free.
+
+## Resources
+
+### Audio
+
+- [PodCTL](https://twitter.com/PodCTL)
+- [Kubernetes Podcast](https://kubernetespodcast.com)
+- [The New Stack Podcasts](https://thenewstack.io/podcasts/)
+- [SoundCloud](https://soundcloud.com/search?q=kubernetes)
```

Comment:
I'm not a fan of an undifferentiated search link. Granted, the hits will most likely be appropriate, but there is no way to ensure that. I'd either strike this, or add the most relevant one(s).

Reply:
Would this be better?
```suggestion
- [SoundCloud](https://soundcloud.com/search/sounds?q=kubernetes&filter.genre_or_tag=technology)
```

If not, I don't mind striking it.

Reply:
No problem. Will strike.